### ISSUE TITLE
Multi client on 1 VPS

### ISSUE THREAD

**Issue by Florent4223** [zehome/MLVPN#130]
Hello,

My title sums up my question but in detail is that on a single VPS, it is possible to connect several clients (in my case raspberry) which each will use several internet connections (4G or other) with MLVPN?

Thanks

**Comment by zehome** [comment]
Yes, but you will need to launch a matching server instance on your VPS.

This can be done by using multiple configurations, one for each client and by launching multiple mlvpn instances using the systemd generator.

**Comment by Florent4223** [comment]
Thank you for the answer.

If I understand correctly, it would be necessary to launch several mlvpn configuration, for exemple mlvpn0.conf , mlvpn1.conf etc,
and: 
mlvpn -v  --user mlvpn -c /etc/mlvpn/mlvpn0.conf
mlvpn -v  --user mlvpn -c /etc/mlvpn/mlvpn1.conf
mlvpn -v  --user mlvpn -c /etc/mlvpn/mlvpn(n).conf

did i understand correctly?


and the client will automatically take the addresses 10.42.42.2, 10.42.42.3 ...?

thanks

**Comment by zehome** [comment]
yes, but naming configurations like /etc/mlvpn/barcelona.conf /etc/mlvpn/mlvpn/paris.conf etc would be better.
IP addresses must be unique, so choose /30 accordingly.

You also need to select unique binding ports on every configuration.

**Comment by Florent4223** [comment]
thanks I try :)

**Comment by ascomputer** [comment]
hello everyone this project is interesting ... I would like to know one thing, if I have a vps with 2 public ip I can tell client 1 to use ip 1 of vps and to client 2 to use ip 2 of vps?
Thank you

**Comment by zehome** [comment]
Basically, yes, you need to use "bind=" for the server endpoint ip address. For the outgoing IP, you'll need to use IPTables accordingly
